tox.ini

```diff
@@ -73,6 +85,16 @@ commands =
     notdoctest: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} or udf" {posargs:} tests
     notudfdoctest: {env:SNOWFLAKE_PYTEST_CMD} -m "{env:SNOWFLAKE_TEST_TYPE} and not udf" {posargs:} tests
     local: {env:SNOWFLAKE_PYTEST_CMD} --local_testing_mode -m "(integ and localtest) or unit or mock_unit" {posargs:} tests
+    # Snowpark pandas commands:
+    snowparkpandasnotdoctest: {env:SNOWFLAKE_PYTEST_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} tests/unit/modin tests/integ/modin
+    # This one only run doctest but we still need to include the tests folder to let tests/conftest.py to mark the doctest files for us
+    snowparkpandasdoctest: {env:SNOWFLAKE_PYTEST_CMD} --durations=20 -m "{env:SNOWFLAKE_TEST_TYPE}" {posargs:} src/snowflake/snowpark/modin/pandas/ tests/unit/modin
+    local: {env:SNOWFLAKE_PYTEST_CMD} --local_testing_mode -m "(integ and localtest) or unit" {posargs:} tests
```

Comment:
duplicate

Reply:
What do you mean by duplicate? I don't think these are duplicates?

Reply:
line 87 has already defined `local`

Reply:
Oh didn't see that, thanks!